HelpSource/Reference/Syntax-Shortcuts.schelp

```diff
@@ -184,6 +268,13 @@ table::
 ## instead of writing: || you can write:
 ## code:: 'mySymbol' :: || code:: \mySymbol ::
 ::
+note::
+The shorthand cannot be used with symbols from operators:
```

Comment:
this is not quite right. valid symbols that can be used with `\` are `[a-zA-Z_][a-zA-Z0-9_]*|[0-9]*`.  in non-regex-speak, they must either be a series of zero or more digits, or a letter or underscore followed by one or more "word" characters (alphanumeric or underscore). (yes, `\` by itself is a valid symbol!)

Reply:
i think the documentation may not even get this right. the limitation about symbols starting with digits is somewhat arbitrary, see #2676 .

Reply:
My point with that note was not to provide a full specification of what works and what doesn't after `\` , but just a common caveat. The full spec (if someone can figure it out) should go in https://doc.sccode.org/Overviews/SymbolicNotations.html which has a "Symbolic notations to define literals/other objects" (that alas also doesn't say anything about this issue.